.github/workflows/nucliadb_telemetry.yml

```diff
@@ -63,6 +63,7 @@ jobs:
         run: |
           pip install -r test-requirements.txt
           pip install -r nucliadb_telemetry/requirements.txt
+          pip install -e nucliadb_protos
```

Comment:
Why we need protos on telemetry ?

Reply:
I'll have to look again. The tests failed with the dependency install: https://github.com/nuclia/nucliadb/actions/runs/4258853818/jobs/7410507308

Reply:
oh yeah, next line down -- `nucliadb_utils` is installed for the tests. Maybe it can be removed.

Reply:
okay, tests use utils, and utils depends on telemetry

it's turtles all the way down :)

Reply:
Okay, I'm going to remove the nucliadb_utils deps